core/src/main/java/com/graphhopper/routing/lm/PrepareLandmarks.java

```diff
@@ -45,17 +43,10 @@
     private final Graph graph;
     private final LandmarkStorage lms;
     private final LMProfile lmProfile;
-    private int defaultActiveLandmarks;
```

Comment:
`defaultActiveLandmarks` has nothing to do with LM preparation, but its only needed when creating the algorithm, so I removed this here.